src/lib389/lib389/plugins.py

```diff
@@ -1532,7 +1532,7 @@ class PAMPassThroughAuthConfigs(DSLdapObjects):
 
     def __init__(self, instance, basedn="cn=PAM Pass Through Auth,cn=plugins,cn=config"):
         super(PAMPassThroughAuthConfigs, self).__init__(instance)
-        self._objectclasses = ['top', 'extensibleObject', 'nsslapdplugin', 'pamConfig']
+        self._objectclasses = ['top', 'extensibleObject', 'pamConfig']
```

Comment:
It seem a bit strange that  nsslapdplugin objectclass must be removed while this entry is a plugin config entry
as there will be trouble if one of the nsslapdplugin mandatory attribute is missing ...

Reply:
But they aren't needed or mandatory. They should never have been added in the first place. These entries are children to the pam passthrough plugin's dn.

Reply:
Ok. now I understand!